src/OperationResponse.php

```diff
@@ -243,7 +298,7 @@ public function getResult()
      */
     public function getError()
```

Comment:
Was there any further discussion on standardizing the error response? After some thought, I lean towards it being the right thing to do - assuming it isn't a big hurdle to jump.

Reply:
The Compute team said that checking the existing of `error` is sufficient, which is what we are doing for OnePlatform. We currently do not have a use case for using the Error Code and Error Message annotations, so I am leaning towards removing them.

> I lean towards it being the right thing to do

Can you explain what you mean by this?

Reply:
We discussed offline but for anyone else reading, compute returns a different message type for an error response. The discussion is around whether we should standardize compute and other possible APIs error responses into the existing status message when feasible.

Reply:
See the comment below - this now returns `Google\Rpc\Status` based on the descriptors for error code and message.

Reply:
The users would still be able to access it by calling `$operationResponse->getLastProtoResponse()->getError()`